Utility/ApiLogUtility.php

```diff
@@ -143,4 +144,32 @@ public function getLogByUuid($uuid)
     {
         return $this->logs[$uuid] ?? $this->em->getRepository(ApiLog::class)->findOneByResponseUuid($uuid);
     }
+
+    /**
+     * Warning: Will be move in v2.X
```

Comment:
Why you don't @deprecate this? And please explicit how we can already be prepared to this future update

Reply:
Also the way it's phrased is not clear. Will it be removed entirely or moved somewhere else?